packages/server-wallet/src/__test__/create-channel-2-wallets.test.ts

```diff
@@ -0,0 +1,125 @@
+import {execSync} from 'child_process';
+
+import {
+  CreateChannelParams,
+  Participant,
+  Allocation,
+  FundingStrategy,
+} from '@statechannels/client-api-schema';
+import {makeDestination} from '@statechannels/wallet-core';
+import {BigNumber, ethers} from 'ethers';
+
+import {defaultConfig} from '../config';
+import {Wallet} from '../wallet';
+
+import {getChannelResultFor, getPayloadFor} from './test-helpers';
+
+const a = new Wallet({...defaultConfig, postgresDBName: 'TEST_A'});
+const b = new Wallet({...defaultConfig, postgresDBName: 'TEST_B'});
+
+beforeAll(async () => {
+  execSync('createdb TEST_A $PSQL_ARGS');
+  execSync('createdb TEST_B $PSQL_ARGS');
+  await Promise.all([a.dbAdmin().migrateDB(), b.dbAdmin().migrateDB()]);
+});
+afterAll(async () => {
+  await Promise.all([a.destroy(), b.destroy()]);
+  execSync('dropdb TEST_A $PSQL_ARGS');
+  execSync('dropdb TEST_B $PSQL_ARGS');
+});
+
+const testCases: FundingStrategy[] = ['Unfunded', 'Direct', 'Ledger', 'Virtual'];
+
+it.each(testCases)(
```

Comment:
We may (or may not) wish to assert differently depending on the funding strategy.

Reply:
@tomclose we discussed a 'fake' funding strategy. Perhaps this is already covered by 'Unfunded'?